head.html

```diff
@@ -1,4 +1,5 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
+<link rel="modulepreload" href="/scripts/plugins/rum.js"/>
```

Comment:
Really necessary ?

Reply:
To some extent, it helps with the loading. Since plugins are loaded async by `scripts.js`, this inherently introduces a delay before the lazy phase kicks in. So preloading the key plugins as part of the initial 6 parallel requests we are allowed speeds it up a notch.

Before:
<img width="953" alt="Screen Shot 2022-10-19 at 12 11 18 PM" src="https://user-images.githubusercontent.com/1235810/196663293-7d0133e5-e759-401f-bd06-2113889418f3.png">

After:
<img width="967" alt="Screen Shot 2022-10-19 at 12 11 30 PM" src="https://user-images.githubusercontent.com/1235810/196663307-61b0ddfa-a7e1-4f80-979b-fd3de2ae5d04.png">